definitions/variable/technology/power-plant.yaml

```diff
@@ -125,86 +127,63 @@
-- Turbine Efficiency|Electricity|Hydro|Pumped Storage:
-    description: Efficiency of turbining for a pumped-storage hydro unit
-    unit: '%'
-    skip-region-aggregation: true
 - Load Factor|Electricity|{Variable Renewable Power Type}:
-    description: Load factors for onshore wind power plants
+    description: Load factors for {Variable Renewable Power Type} power plants
     unit: '%'
     skip-region-aggregation: true
-- Maximum Energy Charge|Electricity|Energy Storage System:
-    description: This is the upper bound that energy storage system could charge energy
-      at time "t".
-    unit: GWh
-- Maximum Charge|Electricity|Energy Storage System:
-    description: Maximum charge when storing energy the energy storage system unit.
-    unit: GW
-- Roundtrip Efficiency|Electricity|Energy Storage System|{Energy Storage System Type}:
+- Roundtrip Efficiency|Electricity|{Electricity Storage Type}:
     description: Efficiency when charging energy from the grid to the system.
-    unit: 
-- Maximum Discharge|Electricity|Energy Storage System:
-    description: Maximum discharge when consuming energy from the energy storage system
-      unit.
-    unit: GW
-- Maximum Storage|Electricity|Energy Storage System:
-    description: Maximum energy that can be stored by the energy storage systems.
-    unit: GWh
-- Maximum Storage|Electricity|Energy Storage System|{Energy Storage System Type}:
-    description: Maximum energy that can be stored by the Lithium Iron Batteries systems.
-    unit: GWh
-- Minimum Storage|Electricity|Energy Storage System:
-    description: Minimum energy that can be stored by the energy storage system unit.
-    unit: GWh
-- Spillage|Electricity|Energy Storage System:
-    description: Energy spillage that could not be stored by an energy storage system
-      (ESS). Several energy models include spillage variables related to renewable
+    unit:
+- Spillage|Electricity|{Electricity Storage Type}:
+    description: Energy spillage that could not be stored by a {Electricity Storage Type} unit. Several energy models include spillage variables related to renewable
       energy production; however, another way is to include a spillage variable to
-      the ESS unit since it is the last sink of the power network.
-    unit: GWh
-- Seasonality|Electricity|Energy Storage System|{Energy Storage System Type}:
-    description: seasonality (daily, weekly, monthly, etc.) of {Energy Storage System Type}
-    unit: [daily,weekly,monthly]
-- Seasonality|Electricity|Hydro|Reservoir:
-    description: seasonality (daily, weekly, monthly, etc.) of Reservoir
-    unit: [daily,weekly,monthly]
-- Seasonality|Electricity|Hydro|Pumped Storage:
-    description: seasonality (daily, weekly, monthly, etc.) of Pumped Storage
-    unit: [daily,weekly,monthly]
+      the {Electricity Storage Type} unit since it is the last sink of the power network
+    unit:
+    - MW
+    - GW
+- Storage Type|Electricity|{Electricity Storage Type}:
```

Comment:
There is a problem with this variable: what will be its value? We should think of sth where we can have a figure in the value column and a uni; this is why I had suggested to have 'days' as the unit ,, the when it is daily the value is 1, when monthly the value is 30 and when weekly the value is 7; + it is too restrictive as there are storages with longer seasonality (eg the reservoirs in norway)